components/script/dom/performanceresourcetiming.rs

```diff
@@ -86,8 +86,8 @@ impl PerformanceResourceTiming {
             redirect_start: 0.,
             redirect_end: 0.,
             fetch_start: fetch_start,
-            domain_lookup_start: 0.,
             domain_lookup_end: 0.,
+            domain_lookup_start: 0.,
```

Comment:
We need to adjust the `from_resource_timing` method below this to initialize the value to the stored one as well.

Reply:
Ok, done :)